src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -40,6 +41,41 @@ public Task Head()
             }
         }
 
+        [RavenAction("/databases/*/docs/$", "GET")]
```

Comment:
Isn't this the exact same thing as `/docs?id=orders/1`, but without the array, in this case?

The reason we removed this endpoint is that it complicates the routes, and it requires us to encode the url, which isn't good.